src/engine/QueryExecutionTree.cpp

```diff
@@ -267,26 +275,12 @@ nlohmann::json QueryExecutionTree::writeJsonTable(
 }
 
 // _________________________________________________________________________________________________________
-void QueryExecutionTree::writeTable(
+http_streams::stream_generator QueryExecutionTree::writeTable(
     const IdTable& data, char sep, size_t from, size_t upperBound,
-    const vector<std::optional<pair<size_t, ResultTable::ResultType>>>&
-        validIndices,
-    std::ostream& out) const {
+    const vector<std::optional<pair<size_t, ResultTable::ResultType>>>
+        validIndices) const {
   shared_ptr<const ResultTable> res = getResult();
 
-  // special case : binary export of IdTable
-  if (sep == 'b') {
-    for (size_t i = from; i < upperBound; ++i) {
-      for (size_t j = 0; j < validIndices.size(); ++j) {
-        if (validIndices[j]) {
-          const auto& val = *validIndices[j];
-          out.write(reinterpret_cast<const char*>(&data(i, val.first)),
-                    sizeof(Id));
-        }
-      }
-    }
-    return;
-  }
```

Comment:
This is unused code, this was never called

Reply:
Thanks for noticing, but please leave it in, it is my code.
I have forgotten to implement the "binary-export" command in the server, I will readd it soon.
I think `co_yield std::string_view{<theOriginalParenWithReinterpretCase>}` should do the right thing.